### PR TITLE
Create provider-planetscale

### PR DESCRIPTION
**Pull request by tanberry**
Stub file for new guide on Prisma with PlanetScale... this stub file is under a new dir called `/providers` that will later include a guide for CockroachDB.

### CODE REVIEW

File: content/300-guides/050-database/25-providers/provider-planetscale [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+This document discusses the concepts behind using Prisma and PlanetScale, explains some important concepts and considerations on the commonalities and differences between PLanetScale and other dtabase providers, and leads you through the process for configuring your application to implement PlanetScale.
+
+Two main benfits of using PlanetScale with Prisma are:
+
+- enterprise scalability
+    - <use Jan's content>
+    
+- support non-blocking schema changes
+    - <use Jan's content>
```
Comment (janpio) [inline content/300-guides/050-database/25-providers/provider-planetscale]: We might want to discuss these further - I kinda copy pasted those together.

From a Prisma perspective we are excited that PlanetScale has unlimited connections as this makes it trivial to use Prisma in a serverless context.

File: content/300-guides/050-database/25-providers/provider-planetscale [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<use Jan's content>
+
+### Differences to consider
+
+<use Jan's content, cover FK since PS doesn't enforce FK, etc>
+
+### Additional considerations
+
+<add note about `db push` instead of `prisma migrate`>
```
Comment (janpio) [inline content/300-guides/050-database/25-providers/provider-planetscale]: I think that is the _real_ content of the guide and what people will want to read to learn how to use Prisma with PlanetScale

File: content/300-guides/050-database/25-providers/provider-planetscale [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## How to implement Planetscale
+
+<to be written, include a ssection for using with the Platform>
```
Comment (janpio) [inline content/300-guides/050-database/25-providers/provider-planetscale]: Note that PlanetScale can be used with Prisma ORM alone, no PDP needed (and actually less useful/needed as with other database providers as DataProxy is not needed)

**Comment by tanberry** [comment]
Closing this one, will create new PR based on convo about IA.

**Pull request closed without merging**